pkg/context/httpresponse.go

```diff
@@ -82,7 +82,7 @@ func (w *httpResponse) SetBody(body io.Reader) {
 	w.body = body
 }
 
-// None uses it currently, keep it for future maybe.
+// OnFlushBody None uses it currently, keep it for future maybe.
```

Comment:
```suggestion
// OnFlushBody adds an HTTP body flushing handler function
```

Reply:
This function has been used in `MemoryCache`, so the original comment is outdated.

Reply:
Does easegress use OnFlushBody  function rigt now?
I confused use it on partial data flush to cliet conn.

Reply:
Got it.

Reply:
* The short answer is Yes, check it out [here](https://github.com/megaease/easegress/blob/main/pkg/filter/proxy/proxy.go#L328)
* In the `response.go`, the comment is ` OnFlushBody registers a callback function on flush body` for `MockedHTTPResponse.OnFlushBody`, maybe we can align it with `MockedHTTPResponse`'s.

Reply:
I will learn it in the future.